app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -32,6 +40,7 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {Gust(weather)}
```

Comment:
Since Gust is a component, we should include it as such: `<Gust weather={weather} />`. `description` is not a React Component but a string variable, which is why it is included the way it is. It might be helpful to read through the React documentation's overview, which explains React Components and props: https://reactjs.org/tutorial/tutorial.html#overview.

Reply:
could be inlined, easiest thing to do I think
```jsx
{ weather.current.windGust > 15 &&
 `, gusty wind at ${weather.current.windGust} mph` }
```

Reply:
Sandipa originally had it inline, but Craig suggested it be pulled out into a component: https://github.com/mojotech/helios2/pull/270#discussion_r660001746

Reply:
I'm okay going back to what it was, creating a a whole component for a single span of un-styled text doesn't seem important.

Or if we just renamed the function so it didn't look like it was supposed to be a Component, that just never made it all the way. I imagine if it was named `getWindGustDescription(weather)` I wouldn't bat an eye.